### PR TITLE
Changes to support RAS Integration

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Implemented the changed required to support the RAS integration as documented in this PR https://github.com/ONSdigital/ons-schema-definitions/pull/30

### How to review 
Are the new values present in the downstream data when launched with the version of launcher that supports these values https://github.com/ONSdigital/go-launch-a-survey/pull/50

### CODE REVIEW

File: app/data_model/session_data.py [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+                 form_type,
+                 period_str,
+                 language_code=None,
+                 survey_url=None,
+                 ru_name=None,
+                 ru_ref=None,
+                 submitted_time=None,
+                 case_id=None,
+                 case_ref=None,
+                 account_service_url=None):
```
Comment (dwyeradam) [inline app/data_model/session_data.py]: We'll need to remember to consider forward compatibility for these new fields when used by the [session_store](https://github.com/ONSdigital/eq-survey-runner/blob/master/app/data_model/session_store.py#L82)

Comment (nealedj) [reply app/data_model/session_data.py]: Just chatted about this. Good point! Let's consider that a separate job and get the session data incompatibility fixed before the next deployment.

Comment (dwyeradam) [reply app/data_model/session_data.py]: Card raised: https://trello.com/c/mQrMxzDw

**Review by dwyeradam — APPROVED** [review]

File: app/views/errors.py [hunk 2]
```diff
@@ -95,14 +96,21 @@ def get_tx_id():
...
+        account_url = cookie_session[ACCOUNT_URL]
+    return account_url
+
+
 def render_template(template_name):
     tx_id = get_tx_id()
     user_agent = user_agent_parser.Parse(request.headers.get('User-Agent', ''))
 
-    return render_theme_template(session.get('theme', 'default'),
+    return render_theme_template(cookie_session.get('theme', 'default'),
```
Comment (nealedj) [inline app/views/errors.py:110]: It looks like there's no test coverage for the `respondent_account_url` getting set by the JWT payload.

**Review by nealedj — CHANGES_REQUESTED** [review]

File: app/views/errors.py (hunk 2 above)
Comment (jonnyshaw89) [reply app/views/errors.py:110]: @nealedj i have added test coverage

File: app/views/errors.py [hunk 3]
```diff
@@ -95,14 +96,21 @@ def get_tx_id():
     return tx_id
 
 
+def get_account_url():
+    account_url = current_app.config['RESPONDENT_ACCOUNT_URL']
+    if ACCOUNT_URL in cookie_session:
```
Comment (nealedj) [inline app/views/errors.py:101]: If `cookie_session[ACCOUNT_URL` is present but null then `None` will be rendered in the `href`. We should check that there is a non-empty value for the URL.

**Review by nealedj — CHANGES_REQUESTED** [review]

**Review by nealedj — APPROVED** [review]

**Pull request merged**